web/app.fluidity.money/app/util/api/graphql.ts

```diff
@@ -11,6 +11,8 @@ type GqlEndpoint = {
 
 type GqlBackend = "hasura";
 
+const HasuraUrl = "http://localhost:8080/v1/graphql";
```

Comment:
dev placeholder?

Reply:
I couldn't get this to work with an environment variable weirdly...